src/patternfly/components/Page/page.scss

```diff
@@ -390,15 +375,17 @@ $pf-page-v5--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
 
 // Sidebar
 .#{$page}__sidebar {
-  @include pf-v5-t-dark;
-
   z-index: var(--#{$page}__sidebar--ZIndex);
   display: flex;
   flex-direction: column;
   grid-area: nav;
   grid-row-start: 2;
   grid-column-start: 1;
   width: var(--#{$page}__sidebar--Width);
+  padding-top: 0;
+  padding-bottom: var(--pf-t--global--spacer--lg);
+  padding-left: var(--pf-t--global--spacer--lg);
```

Comment:
a bunch of these styles using tokens directly that we'll need component vars for